NeoMathEngine/src/CPU/CpuMathEngineDnn3dConv.cpp

```diff
@@ -143,7 +143,7 @@ void CCpuMathEngine::blob3dConvolution1x1x1LearnAdd( const CCommon3dConvolutionD
 				for( int i = 0; i < inputBlob.Width(); ++i ) {
 					CConstFloatHandle inputPixelData = inputColData;
 					for( int k = 0; k < inputBlob.Depth(); ++k ) {
-						vectorCopy( inputBlobDataPtr, inputPixelData, input.Channels() );
+						vectorCopy( GetRaw( inputBlobDataPtr ), GetRaw( inputPixelData ), input.Channels() );
```

Comment:
change `inputBlobDataPtr` and `inputPixelData` types to `float*` and remove `GetRaw` calls from here (it's inside of 4 `for` cycles)